src/thermompnn/datasets/v2_datasets.py

```
@@ -1,21 +1,32 @@
-from Bio import pairwise2
-from tqdm import tqdm
+import pickle
 from copy import deepcopy
 from itertools import permutations
-import itertools
+
+import numpy as np
+import pandas as pd
+import torch
+from Bio import pairwise2
 from sklearn.preprocessing import minmax_scale
```

Ruff (0.8.2)

`sklearn.preprocessing.minmax_scale` imported but unused

Remove unused import: `sklearn.preprocessing.minmax_scale`

(F401)